examples/zaps-nwc.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config()
```

Comment:
don't require dotenv here, this is not a dependency 

you can just run this example with NOSTR_PRIVATE_KEY=foo NWC_URL=bar node examples/zaps-nwc.js